### DISCUSSION TITLE
SQL & Send Email (SMTP)

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#4619] [Help]
Hello,

I have a SQL Query connected to our Active Directory. 
If you're wondering how the Query looks:

{
  "WindowsLogin": "xyz",
  "Location": "xyz",
  "Corporation": "xyz",
  "PNR": 123,
  "Listed": "xyz"
}

When I add {{ steps.1.info }} to my Automation it works as well but the Email I get looks like this:
![bug](https://user-images.githubusercontent.com/95284278/155091705-aa0b9004-64bb-4fd9-80f8-aad2e25a15e9.png)

How or when can I fix the output of the Email?

Thanks!

**Comment by mjashanks** [comment]
So it looks like you probably want to use some of the fields within your email. Here's an example of what you could write

```
Hello,

There was a login attempt by {{ steps.1.info.0.WindowsLogin }} from {{ steps.1.info.0.Corporation }}. 

Thank you
```

I have no idea what the context of this data should be, but hopefully that gives you an idea of how to use it!

**Reply by kasgithub8** [reply]
Thanks @mjashanks,

that worked perfectly. Is there a way to insert a line break after 1 object?

**Reply by mjashanks** [reply]
Sorry - I'm not sure what you mean... can you not just type a paragraph with line breaks as I did above?

Can you give an example of what you want the output to look like?

**Reply by kasgithub8** [reply]
Sure, 
I want it to look like this:

{{ steps.1.info.0.WindowsLogin }}
{{ steps.1.info.0.Corporation }}

If I insert this into my HTML Contents, the output looks like this:

{{ steps.1.info.0.WindowsLogin }} {{ steps.1.info.0.WindowsLogin }}

**Reply by mjashanks** [reply]
Ah, i see.

in that case, you might need to do something like 

```
{{ steps.1.info.0.WindowsLogin }}
<br>
{{ steps.1.info.0.Corporation }}
```

... as it is expecting **html** contents. You might want to use an HTML email editor to help you build this properly - I'm sure there are loads of tools out there for this, though I've not used one!

**Reply by kasgithub8** [reply]
Uh, now it works. Thank you vm!

**Reply by kasgithub8** [reply]
Hello @mjashanks,

I have another question about this. When I wanna send a "Created Row" per email, how can I split the columns like you did above? Because we only have "Trigger.Row", "Trigger.Id" & "Trigger.Revision".

**Reply by mjashanks** [reply]
You can use `{{ Trigger.Row.ColumnName }}`
